docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,188 @@
+💡 If you only need to override the documentation page for a single component, we recommend creating an MDX file and referencing it directly via the `<Meta of={} />` Doc Block.
+
+</div>
+
+## Setup custom documentation
+
+Storybook provides support for MDX 2, an open standard markup language, combining two other ones: Markdown, which is used for formatting text, and JSX, which is used for rendering dynamic elements on a page. To enable custom documentation for your stories with this format, start by updating your Storybook configuration file (i.e., `.storybook/main.js|ts|cjs`).
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/custom-docs-page.mdx.mdx',
-    'common/button-story-docspage-with-mdx.js.mdx',
+    'common/storybook-auto-docs-main-mdx-config.js.mdx',
+    'common/storybook-auto-docs-main-mdx-config.ts.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### With a custom component
-
-Storybook's UI is built using React. If you want to include a custom component to display documentation, you'll need to update your environment to allow React components to be correctly transpiled.
-
-For example, with Angular start by adding a `babel.config.js` file at the root of the project with the following content:
-
-```js
-// babel.config.js
-
-module.exports = function (api) {
-  process.env.NODE_ENV === 'development' ? api.cache(false) : api.cache(true);
-  const presets = [
-    [
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current',
-        },
-      },
-    ],
-    '@babel/preset-typescript',
-    '@babel/preset-react',
-  ];
-  const plugins = [];
-  return {
-    presets,
-    plugins,
-  };
-};
-```
-
-Then, update your `tsconfig.json` to include the following:
-
-```json
-{
-  "compilerOptions": {
-    ....
-    "allowJs": true,
-    "jsx": "react-jsx",
-  },
-}
-```
-
-Finally write your custom React component and update the `docs.page` [parameter](../writing-stories/parameters.md) to render the custom documentation.
+Create an MDX file for your component in the same directory as your stories and components to add your custom documentation. The file should match your component's name (e.g., `Button.mdx`).
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/custom-docs-page.js-component.js.mdx',
-    'common/custom-docs-page.ts-component.ts.mdx',
-    'common/button-story-docspage-with-custom-component.js.mdx',
+    'common/storybook-auto-docs-baseline-example.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-## Remixing DocsPage using doc blocks
+Once the MDX documentation is loaded, Storybook will render it alongside your component's story overriding any existing documentation enabled via the [`tags`](#setup-automated-docs) configuration property.
```

Comment:
We are actually going to add a warning if you set the tag but then override the auto doc with an MDX file: https://linear.app/chromaui/issue/SB-1032/warn-if-setting-tags-[docspage]-and-then-overriding-with-meta-of=

So I'm wondering if this should explicitly say you should drop the tag if you create an MDX file.